admin/orders.php

```diff
@@ -721,43 +721,37 @@ function couponpopupWindow(url) {
         <div class="row noprint"><?php echo zen_draw_separator('pixel_trans.gif', '1', '5'); ?></div>
         <div class="row noprint">
           <div class="formArea">
-              <?php echo zen_draw_form('status', FILENAME_ORDERS, zen_get_all_get_params(array('action')) . 'action=update_order', 'post', 'class="form-horizontal"', true); ?>
+              <?php echo zen_draw_form('statusUpdate', FILENAME_ORDERS, zen_get_all_get_params(array('action')) . 'action=update_order', 'post', 'class="form-horizontal"', true); ?>
             <div class="form-group">
                 <?php echo zen_draw_label(TABLE_HEADING_COMMENTS, 'comments', 'class="col-sm-3 control-label"'); ?>
               <div class="col-sm-9">
-                  <?php echo zen_draw_textarea_field('comments', 'soft', '60', '5', '', 'class="form-control"'); ?>
+                  <?php echo zen_draw_textarea_field('comments', 'soft', '60', '5', '', 'id="comments" class="form-control"'); ?>
               </div>
             </div>
             <div class="form-group">
                 <?php echo zen_draw_label(ENTRY_STATUS, 'status', 'class="col-sm-3 control-label"'); ?>
               <div class="col-sm-9">
-                  <?php echo zen_draw_pull_down_menu('status', $orders_statuses, $order->info['orders_status'], 'class="form-control"'); ?>
+                  <?php echo zen_draw_pull_down_menu('status', $orders_statuses, $order->info['orders_status'], 'id="status" class="form-control"'); ?>
               </div>
             </div>
             <div class="form-group">
-                <?php echo zen_draw_label(ENTRY_NOTIFY_CUSTOMER, 'notify', 'class="col-sm-3 control-label"'); ?>
+                <div class="col-sm-3 control-label" style="font-weight: 700;"><?php echo ENTRY_NOTIFY_CUSTOMER; ?></div>
```

Comment:
Why remove the label, and introduce a style="" ?

Reply:
This is defined as a label, but a label can be linked to only one input.
In this case it is covering three radio buttons so it cannot be a label for three inputs, it is really only a title for the group. Adding the style to make it bold as all the rest is messy I know, but the bold attribute is applied to the other in that section through the label tag, so I didn't see another way to make it bold apart from copying that particular style.

Reply:
That sounds sensible 😃